things/api.py

```diff
@@ -498,7 +498,7 @@ def today(**kwargs):
     might not be up to date anymore if you didn't open the app recently.
     To get around this limitation, we here make a prediction of what
     to-dos would show up in Today if you were to open the app right now.
-    This prediction does currently not include repeating tasks.
+    This prediction does currently not include repeating to-dos.
```

Comment:
We earlier wrote: `"task" is a technical term used in the database to refer to a to-do, project, or heading.". This method could also return a project, therefore `task` is the correct term, isn't it?

Reply:
I used "to-do" several time in this docstring where "task" would have been the correct term. Good catch.

As for repeats: since only to-dos can repeat, I think this is the more specific term to use here. (My rule: be as specific as possible unless it hurts readability.)

Reply:
Actually, you can have repeating projects:
1. Open Things
2. Go to an area
3. Create new project
4. Click on `...` next to the title
5. Select `Repeat...`

Reply:
👌  You're right. I learned something new. Thanks. :)